src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule713atclauses/InputIncorrectAtClauseOrderCheck1.java

```diff
@@ -11,109 +11,110 @@
  * @see Some javadoc.
  * @author max
  */
-class InputIncorrectAtClauseOrderCheck1 implements Serializable
-{
-    /**
-     * The client's first name.
-     * @serial
-     */
-    private String fFirstName;
+class InputIncorrectAtClauseOrderCheck1 implements Serializable {
+  /**
+   * The client's first name.
+   *
+   * @serial
+   */
+  private String fFirstName;
 
-    /**
-     * The client's first name.
-     * @serial
-     */
-    private String sSecondName;
+  /**
+   * The client's first name.
+   *
+   * @serial
+   */
+  private String sSecondName;
 
-    /**
-     * The client's first name.
-     * @serialField
-     */
-    private String tThirdName;
+  /**
+   * The client's first name.
+   *
+   * @serialField
+   */
+  private String tThirdName;
+
+  /**
+   * Some text.
+   *
+   * @param aString Some text.
+   * @return Some text.
+   * @serialData Some javadoc.
+   * @deprecated Some text.
+   * @throws Exception Some text. // violation 'Block tags have to appear in the order .*'
+   */
+  String method(String aString) throws Exception {
+    return "null";
+  }
 
+  /**
+   * Some text.
+   *
+   * @serialData Some javadoc.
+   * @return Some text.
+   * @param aString Some text. // violation 'Block tags have to appear in the order .*'
+   * @throws Exception Some text.
+   */
+  String method1(String aString) throws Exception {
+    return "null";
+  }
+
+  /**
+   * Some javadoc.
+   *
+   * @version 1.0
+   * @since Some javadoc.
+   * @serialData Some javadoc.
+   * @author max
+   */
+  class InnerClassWithAnnotations1 {
     /**
      * Some text.
-     * @param aString Some text.
+     *
      * @return Some text.
-     * @serialData Some javadoc.
      * @deprecated Some text.
-     * @throws Exception Some text. //warn
+     * @param aString Some text. // violation 'Block tags have to appear in the order .*'
+     * @throws Exception Some text. // violation 'Block tags have to appear in the order .*'
```

Comment:
why we have bunch of such meanigful updates in Inputs?
better to keep them as is, and just place violation comment were it is required.

Reply:
reverted. Idk how how this code got formatted automatically. Thanks for pointing this out

Reply:
>formatted automatically.

By what ?